content/get-started/index.md

```diff
@@ -0,0 +1,78 @@
+| Galaxy 101 |   | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-intro-101-everyone/tutorial.html){:target="_blank"}   | [<i class="fas fa-video" aria-hidden="true"></i>](https://gallantries.github.io/video-library/videos/introduction/galaxy-intro-101/tutorial/){:target="_blank"}    |   |
+| Your first Galaxy Analysis | [<i class="fab fa-slideshare" aria-hidden="true"></i>](bit.ly/smorg-2021-intro){:target="_blank"}  |   |[<i class="fas fa-video" aria-hidden="true"></i>](https://youtu.be/1UPCrQxGwcU){:target="_blank"} |   |
+| Galaxy Tour  |   |   |   | [<i class="fas fa-mouse" aria-hidden="true"></i>](https://usegalaxy.org/tours/core.galaxy_ui){:target="_blank"} |
+| Galaxy Histories |   | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/history/tutorial.html){:target="_blank"}  |   | [<i class="fas fa-mouse" aria-hidden="true"></i>](https://usegalaxy.org/tours/core.history){:target="_blank"} |
+| How to reproduce published Galaxy Analysis |  | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-reproduce/tutorial.html){:target="_blank"} |   |   |
+|Introduction to the GTN |[<i class="fab fa-slideshare" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/contributing/slides/introduction.html#1
+){:target="_blank"} |   | [<i class="fas fa-video" aria-hidden="true"></i>](https://www.youtube.com/watch?v=lDqWxzWNk1k){:target="_blank"} |   |
+| How to contribute using Gitpod |   | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/gitpod/tutorial.html){:target="_blank"}  | [<i class="fas fa-video" aria-hidden="true"></i>](https://youtu.be/_-wnsPttBCo){:target="_blank"}  |   |
+|How to create a new tutorial in the GTN   |   |[<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial/tutorial.html){:target="_blank"}   |[<i class="fas fa-video" aria-hidden="true"></i>](https://www.youtube.com/watch?v=Vw9h5kvRH2k){:target="_blank"}  |   |   |
+
+### Useful links
+- [FAQs quickstart](https://training.galaxyproject.org/training-material/faq)
+  
+  Feel like you might have a question that many others have had as well? Check our Frequently Asked Questions quickstart!
+
+- [Public Galaxy Servers](https://galaxyproject.org/use/)
+  
+  List of all Public Galaxy Servers available to use or deploy your own version.
```

Comment:
I would put it before Tutorials. To follow a tutorial you need to understand that there are different servers. 
We could explain this and tell newcomers that they will have to "choose" a server (and provide guidance depending on what they are willing to do, etc.).

Reply:
Okay, after reading this comment maybe it'd be better to divide the links under "Useful links" into different "items" like this two?
So it's not as confusing?